src/components/swapv2/SingleTokenInfo.tsx

```diff
@@ -150,18 +179,25 @@ const SingleTokenInfo = ({
 
   const toggleSeeMore = () => setShowMoreDesc(isSeeMore ? SeeStatus.SEE_LESS : SeeStatus.SEE_MORE)
 
+  const symbol = currency?.symbol
+  const currencyName = tokenInfo.name || currency?.name
+
   return (
     <Wrapper borderBottom={borderBottom}>
-      <Flex>
-        <CurrencyLogo currency={currency} size="24px" />
-        <AboutText>About {currency?.symbol}</AboutText>
+      <Flex alignItems="center">
+        <CurrencyLogo currency={currency} size="24px" style={{ marginRight: 10 }} />
+        <AboutText>
+          About {symbol} {currencyName !== symbol ? `(${currencyName})` : null}
```

Comment:
Wrap trans

Reply:
I will update, but not now, because description 100% in English.  I will update when this feature support all language.